draft-ietf-jsonpath-base.md

```diff
@@ -556,7 +556,8 @@ A JSONPath query consists of a sequence of selectors. Valid selectors are
   * Array slice selector `[<start>:<end>:<step>]`, where the optional
     values `<start>`, `<end>`, and `<step>` are integer literals.
   * Nested descendants selector `..`.
-  * Union selector `[<sel1>,<sel2>,...,<selN>]`, holding a comma delimited list of index, index wild card, array slice, and filter selectors.
+  * List selector `[<sel1>,<sel2>,...,<selN>]`, holding a comma
+    separated list of index and slice selectors.
```

Comment:
I think calling it that it only hold index and slice selectors is good.  During the meeting, I feared that it would merely a "collection of selectors," implying that it could itself hold a list selector, which would be recursive and confusing.

Do we need to add "member names" as is mentioned on line 966?

Reply:
index selectors can contain quoted member names, so that should not be necessary (after we have called it out already).